howso/direct/client.py

```diff
@@ -4025,6 +4028,29 @@ def get_marginal_stats(
         ----------
         trainee_id : str
             The ID of the Trainee to retrieve marginal stats for.
+        condition : dict or None, optional
+            A condition map to select which cases to compute marginal stats
+            for.
+
+            .. NOTE::
+                The dictionary keys are the feature name and values are one of:
+
+                    - None
+                    - A value, must match exactly.
+                    - An array of two numeric values, specifying an inclusive
+                      range. Only applicable to continuous and numeric ordinal
+                      features.
+                    - An array of string values, must match any of these values
+                      exactly. Only applicable to nominal and string ordinal
+                      features.
+        num_cases : int, default None
+            The maximum amount of cases to use to calculate marginal stats.
+            If not specified, the limit will be k cases if precision is
+            "similar". Only used if `condition` is not None.
+        precision : str, default None
```

Comment:
I wonder if there should be a mechanic that if something other than `{None, "exact", "similar"}` is provided, it defaults to "exact", but raises a warning since `howso-engine` will do similar, but we won't have an opportunity to raise the warning.

Reply:
I'm leaning towards raising a ValueError if 'condition' is specified as a string that is not either "exact" or "similar". What do you think?

Reply:
I would probably add that logic to every method in this file that uses 'precision'.

Reply:
Let's stick with a warning.

Reply:
After discussion in person - we are going to go with warnings.